Starship/Assets/Resources/Localization/English/Quests.xml

```diff
@@ -101,8 +100,7 @@
     <string name="HappyEaster_Message1">Hey captain! Long time no see! Have an egg-celent Easter! Here is your egg.</string>
     <string name="HappyEaster_Message2">Hey captain! Don\'t worry, nothing has happened. I just want to wish you happy Easter. Here is your egg.</string>
     <string name="HappyEaster_Message3">Greetings, Commander. I have a delivery for you. Have an egg-celent Easter!</string>
-
-    <string name="Quest_Covid">COVID-6019</string>
+    <string name="Quest_Covid">Invasion</string>
```

Comment:
Why the name change?

Reply:
Doesn't reflect the quest in my opinion. COVID hype is over and I dunno why to leave its reference here.

Reply:
Well, that's not for us to decide. I can change it in EHCE but not in vanilla